pkg/images/extractor.go

```diff
@@ -0,0 +1,104 @@
+package images
```

Comment:
this package would be better named `metadata` to be agnostic to logic for extracting images and fqdns